tests/unit/CompanyProvisions.spec.ts

```diff
@@ -46,7 +46,7 @@ describe('company provisions', () => {
       .toBe('This company has Pre-existing Company Provisions.')
     expect(wrapper.find(changeCompanyProvisionsButton).exists()).toBe(true)
     expect(wrapper.find(defineCompanyProvisionsTitle).text())
-      .toBe('Pre-existing Company Provisions')
+      .toBe('Pre-existing  Company Provisions')
```

Comment:
Ah, this is because of the br in the label.
This is why I something use spans for individual lines -- so I can use toBe() instead of toContain() -- though in this case you seem to be getting away with it 🤞

Reply:
Yea, i threw in the break to meet the designs. I know it was outside of my components but it looked nice to be uniform.
Thats a good idea though, i really like spans as well. Narrow down on those elements!